pyclock.py

```diff
@@ -7,7 +7,13 @@
 from os import environ
 
 
-__description__ = ' A digital clock for the terminal '
+__program__ = 'PyClock'
+__version__ = '0.0.0'
+__description__ = 'A digital clock for the terminal'
+__author__ = 'Brandon Dreager'
+__copyright__ = 'Copyright (c) 2016 Brandon Dreager'
+__license__ = 'MIT'
+__website__ = 'https://github.com/Regaerd/PyClock'
```

Comment:
It's fine for both cases. But I will prefer them stay with the main script.

A good example is if you want to `--version`, you end up having a version number in two places, and I will bet there is a good chance the one in main script is forgotten. So, the package ends up with right version number on PyPI, but after the installation, you get the wrong number from the main script. Of course, the script can always query Python package database. It simply is error-prone in my opinion if put in `setup.py`.

Another good practice, at least for version number, is to have `version.py`.

Nonetheless, if you prefer having them in `setup.py`, then I will move them, I don't think duplicate them there is a good idea, the reason is as same as the version above. As times goes, one would be outdated.